libs/sdk-ui-kit/styles/scss/header.scss

```diff
@@ -804,19 +804,21 @@ button.gd-button-primary {
     }
 }
 
-.gd-upsell-button {
+button.gd-button-primary.gd-upsell-button {
     width: 93px;
     height: 27px;
     font-style: normal;
     font-weight: normal;
     font-size: 14px;
     margin: auto 7px;
     padding: 0;
+    border: 1px solid transparent;
     line-height: 14px;
     text-align: center;
     border-radius: 3px;
     color: #fff;
     background: #8351a4;
+    font-family: "Avenir", sans-serif;
```

Comment:
Do we really need to define `font-family` for this button? If we do need, I think we should add `gdcustomfont` before `avenir`, as we are doing in other places in the SDK https://github.com/search?q=org%3Agooddata+gdcustomfont&type=code